### PR TITLE
Cleanup and update of ET navigation demo and scripts

### PR DESCRIPTION
**Pull request by sostel**
Overview
---
- Removed some deprecated parameters in the scroll scripts that were still shown in the Editor.
- Updated hand zoom parameter that was specific to zoom, but was shown for scroll.

### CODE REVIEW

File: Assets/MixedRealityToolkit.Examples/Demos/EyeTracking/Demo_ScrollPanZoom/Scripts/BaseClasses/PanZoomBase.cs [hunk 1]
```diff
@@ -121,7 +121,7 @@ public abstract class PanZoomBase : MonoBehaviour,
         private Vector3 navPos = Vector3.zero;
         private bool isFocused = false;
         internal bool isZooming = false;
-        public bool handZoomEnabledOnStartup = false;
+        internal bool ZoomGestureEnabledOnStartup = false;
```
Comment (davidkline-ms) [inline Assets/MixedRealityToolkit.Examples/Demos/EyeTracking/Demo_ScrollPanZoom/Scripts/BaseClasses/PanZoomBase.cs]: curious as to why this changed to internal

File: Assets/MixedRealityToolkit.Examples/Demos/EyeTracking/Demo_ScrollPanZoom/Scripts/PanZoom_RectTransf.cs [hunk 2]
```diff
@@ -70,6 +70,10 @@ public class PanZoom_RectTransf : PanZoomBase_RectTransf
         [SerializeField]
         private MixedRealityInputAction Zoom_Gesture = MixedRealityInputAction.None;
 
+        [Tooltip("Enable or disable hand gestures for zooming on startup.")]
+        [SerializeField]
+        private bool Zoom_GestureEnabledOnStartup = false;
```
Comment (davidkline-ms) [inline Assets/MixedRealityToolkit.Examples/Demos/EyeTracking/Demo_ScrollPanZoom/Scripts/PanZoom_RectTransf.cs]: the pattern for MRTK has been to use lower case private field names and not using underscores.

File: Assets/MixedRealityToolkit.Examples/Demos/EyeTracking/Demo_ScrollPanZoom/Scripts/BaseClasses/PanZoomBase.cs (hunk 1 above)
Comment (sostel) [reply Assets/MixedRealityToolkit.Examples/Demos/EyeTracking/Demo_ScrollPanZoom/Scripts/BaseClasses/PanZoomBase.cs]: The PanZoomBase class is consumed both by the scroll as well as the pan-and-zoom scripts. The scroll scripts don't use zoom though and hence it doesn't make sense to show this parameter in the Unity Editor.

File: Assets/MixedRealityToolkit.Examples/Demos/EyeTracking/Demo_ScrollPanZoom/Scripts/PanZoom_RectTransf.cs (hunk 2 above)
Comment (sostel) [reply Assets/MixedRealityToolkit.Examples/Demos/EyeTracking/Demo_ScrollPanZoom/Scripts/PanZoom_RectTransf.cs]: Good to know . I guess then I have to start a larger cleanup.

Comment (davidkline-ms) [reply Assets/MixedRealityToolkit.Examples/Demos/EyeTracking/Demo_ScrollPanZoom/Scripts/PanZoom_RectTransf.cs]: I just noticed this is in a demo script. while the standards should be the same, I do not believe the team is quite a strict :)

Comment (sostel) [reply Assets/MixedRealityToolkit.Examples/Demos/EyeTracking/Demo_ScrollPanZoom/Scripts/PanZoom_RectTransf.cs]: Nah, this is a valid ask. We were thinking of moving these over to core at some point as they seem to be useful to the community. I'll update it.

Comment (sostel) [reply Assets/MixedRealityToolkit.Examples/Demos/EyeTracking/Demo_ScrollPanZoom/Scripts/PanZoom_RectTransf.cs]: Quick follow up question: Entirely lowercase? Or just the beginning of the name lowercase?

Comment (davidkline-ms) [reply Assets/MixedRealityToolkit.Examples/Demos/EyeTracking/Demo_ScrollPanZoom/Scripts/PanZoom_RectTransf.cs]: just the beginning :)

**Pull request merged**